recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+# applicable metal framework only after 11: 
+# see this: https://github.com/phracker/MacOSX-SDKs/search?q=%3CMetal%2FMTLBlitCommandEncoder.h%3E
+# also some symbols only available after 11.0
+# Please consult conda-forge/core before doing this
```

Comment:
You should have consulted conda-forge/core before doing this.

Reply:
Ooops, I just saw your comment. I am really bad at following through these comments (like you saw in the other PR). Sorry, so let's correct that now and I will do better going foward.

@conda-forge/core: this likely will be an issue with metal going forward. Sorry I went ahead and merged with consulting as I was excited about the implementation. Any ideas/thoughts would be appreciated and please see conversation in https://github.com/conda-forge/conda-forge.github.io/issues/1537 for more context.

Reply:
(I need to have me an isuruf warning to get my attention whenever you comment on my stuff. I somehow miss a lot of this as my notifications settings are totally messed up...)

Reply:
Also, note the link in the comment is just the start of the debugging. Upstream they specify Metal version 2.3 or something that only works for 11+ (I will try to get the exact link next time we have a PR here)

Reply:
This is incomplete. Please see the docs about `__osx` run requirement. You'll have to mark this build as broken after you have fixed it.

Reply:
> This is incomplete. Please see the docs about `__osx` run requirement. You'll have to mark this build as broken after you have fixed it.

I saw that part of the documentation and my reading from it was that the `__osx` part was optional. And it made sense because the error you would get once you install tvm-py and import it is something along the lines of "you need sdk 11.0 or higher". 

```
OSError: dlopen(/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/libtvm.dylib, 10): Symbol not found: _BNNSFilterApplyTwoInputBatch
  Referenced from: /Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/libtvm.dylib (which was built for Mac OS X 11.0)

```

Did I misread this? Should I still include that?

Full error:

```
(testtvms) ngam@mp15:~$ python
Python 3.10.2 | packaged by conda-forge | (main, Mar  8 2022, 16:02:23) [Clang 11.1.0 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import tvm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/python3.10/site-packages/tvm/__init__.py", line 26, in <module>
    from ._ffi.base import TVMError, __version__, _RUNTIME_ONLY
  File "/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/python3.10/site-packages/tvm/_ffi/__init__.py", line 28, in <module>
    from .base import register_error
  File "/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/python3.10/site-packages/tvm/_ffi/base.py", line 71, in <module>
    _LIB, _LIB_NAME = _load_lib()
  File "/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/python3.10/site-packages/tvm/_ffi/base.py", line 57, in _load_lib
    lib = ctypes.CDLL(lib_path[0], ctypes.RTLD_GLOBAL)
  File "/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/python3.10/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/libtvm.dylib, 10): Symbol not found: _BNNSFilterApplyTwoInputBatch
  Referenced from: /Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/libtvm.dylib (which was built for Mac OS X 11.0)
  Expected in: /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate
 in /Users/ngam/.Mambaforge-MacOSX-x86_64/envs/testtvms/lib/libtvm.dylib

```

Reply:
Also, it doesn't fail with the target unspecific btw. It just gives warnings. Maybe worth trying this with a lower target anyway to see how that goes?